src/gilded_rose.js

```diff
@@ -46,14 +46,16 @@ function mutateItemSellIn(currentSellIn, amount = 1) {
 }
 
 export function updateQuality(items) {
+  const result = []
   for (var i = 0; i < items.length; i++) {
     if (items[i].name === 'Sulfuras, Hand of Ragnaros') {
       break;
     } else if (items[i].name === 'Aged Brie') {
         if (items[i].sell_in < 0) {
-          items[i].quality = mutateItemQuality(items[i].quality, 2);
+          result.push({...items[i], quality: mutateItemQuality(items[i].quality,2)})
         } else {
-          items[i].quality = mutateItemQuality(items[i].quality);
+          result.push({...items[i], quality: mutateItemQuality(items[i].quality)})
+          // items[i].quality = mutateItemQuality(items[i].quality);
```

Comment:
Wasn't this line deleted in the previous commit?

Reply:
not sure what's going on. here as this isn't on my current working tree. I'll check my branch commits again.